packages/typed-message/dom/Renderer/utils/TransformContext.ts

```diff
@@ -0,0 +1,15 @@
+import type { Transformer, TypedMessage } from '../../../base'
+import { createContext, useContext, useMemo } from 'react'
+import { emptyTransformationContext, TransformationContext } from '../../../base/transformer/context'
```

Comment:
```suggestion
import { emptyTransformationContext, TransformationContext } from '../../../base'
```

Leave details behind some folder, and organize our modules in a higher view.